support/builder/s3migrate.sh

```diff
@@ -197,23 +172,26 @@ setRegion() {
 
 putArtifacts() {
     # handles the literal upload of artifacts to the configured s3 bucket
-    BODY="${1}"
+    DIR=${1}
     BUCKET="${2}"
     REGION="${3}"
-    KEY="${4}"
 
     # upload shellout
     if [  "${s3type}" == 'minio' ]; then
         echo ""
-        aws --endpoint-url "${REGION}" s3 cp "${BODY}" s3://"${BUCKET}/${KEY}"
+        s3-bulk-upload -endpoint="${REGION}" -sourcedir="${DIR}" -bucket="${BUCKET}"
     else
         echo ""
-        aws s3api put-object --bucket "${BUCKET}" --key "${KEY}"  --body "${BODY}" --region "${REGION}"
+        s3-bulk-upload -backend="aws" -region="${REGION}" -sourcedir="${DIR}" -bucket="${BUCKET}"
     fi
 }
 
 genS3Config() {
-    aws configure
+    echo ""
+    read -r -p "AWS Access Key ID [DEFAULTKEYID]: " access_key_id
```

Comment:
Are these defaults in brackets correct and do they get applied if the user just hits enter?

Reply:
These defaults _used_ to be correct but they aren't after our tweaks from friday, let me update this so that both of these things are true.

Reply:
Actually, the thing about changing these is that this function gets used both for S3 as well as minio. I originally had these DEFAULTKEYID and DEFAULTSECRETACCESSKEY because they more closely mirrored what an expected string for the s3 side of things would look like. Ergonomically does it make more sense for us to keep these this way, or should I reconfigure to match the rest of the code?

Reply:
Right, I'm not sure how the default values here actually get set.. you could just remove them, I think that would be fine

Reply:
Yeah I think thats the right thing. Better not to confuse people, the defaults _can_ get set quite easily but from a UX perspective that might add further confusion